### DISCUSSION TITLE
Image is shown in kitty untill I close the Neovim

### DISCUSSION THREAD

**Discussion by Demianeen** [3rd/image.nvim#107] [Q&A]
I istalled this awasome plugin to be able to view images, but I found myself unable to close images without quitting Neovim:
https://github.com/3rd/image.nvim/assets/51330172/b1109e13-76cd-4491-98d8-5081e0ac4dad

Do somebody know if it is possible to close images without quitting neovim?

**Comment by 3rd** [comment]
@Demianeen Hey, need some details on your setup. Are you using Kitty?
Wasn't able to reproduce the issue with Kitty, might be some ueberzug issue if you're using that.
Are you running recent versions of everything?

https://github.com/3rd/image.nvim/assets/59587503/c927bf66-5275-4d8f-ae74-4899f73b4146

**Comment by Demianeen** [comment]
Hi @3rd, great to hear that this is not intended behaviour!

Yep, I use kitty 0.31.0, M1 chip and MacOS 14.2.1 (23C71).


I also have very simple kitty configuration, consisting only from color and font settings:
```
:     font_family      Hark Nerd Font Mono
:     bold_font        auto
:     italic_font      auto
:     bold_italic_font auto
# font_size 11.0


# Color Scheme {{{
foreground #c0caf5
background #222435
selection_foreground #c0caf5
selection_background #33467c
url_color #73daca
bold #eeeeee

# Cursor
cursor #c0caf5
cursor_text_color #15161e

#: The color table {{{
# there are normal and brighter version of the color

#: black
color0 #000000
color8 #414868

#: red
color1 #f7768e
color9 #f7768e

#: green
color2  #9ece6a
color10 #9ece6a

#: yellow
color3  #e0af68
color11 #e0af68

#: blue
color4  #7aa2f7
color12 #7aa2f7

#: magenta
color5  #bb9af7
color13 #bb9af7

#: cyan
color6  #7dcfff
color14 #7dcfff

#: white
color7  #a9b1d6
color15 #c0caf5
```

**Reply by Demianeen** [reply]
Hi @3rd! Do you think it is the issue with M1?

**Reply by 3rd** [reply]
Hey, maybe it's some weird interaction with your file tree setup, does it have any weird issues like that if you don't use it?

**Reply by Demianeen** [reply]
Not really, I use Lazyvim setup with [Lazyvim's default neo-tree settings](https://www.lazyvim.org/plugins/editor#neo-treenvim) except keymaps. 

Can you try if you have the same issue with my [config](https://github.com/Demianeen/lazyvim/tree/delete-me)?

**Reply by 3rd** [reply]
Update and try again please :pray:

**Reply by Demianeen** [reply]
I'm so sorry for the silly question, it seems like my Magick library has suddenly stopped being recognized by image.nvim.

https://github.com/3rd/image.nvim/assets/51330172/ba0f153c-a749-4cca-b9f5-e91ed662a66a

Do you see anything I might be doing wrong?

**Reply by 3rd** [reply]
Hey, disable image.nvim, open nvim and run require("magick"), it should show you the error.

**Reply by nfjdu** [reply]
having similar problem, moving from .md file to netrw or switching between nvim tabs: 

https://github.com/3rd/image.nvim/assets/43147143/fdaefbb2-4cb1-48fe-96d6-a74034016100

is there a way to deal with it? like force some redraw end etc.?
